api/WebGLVertexArrayObjectOES.json

```diff
@@ -30,7 +30,7 @@
             "version_added": "14"
           },
           "safari": {
-            "version_added": false
+            "version_added": "15"
```

Comment:
This is the only change I've spotted so far without an iOS pairing. Is this intentional?

Reply:
This is not intentional and will be reverted.